pkg/models/sidecar.go

```diff
@@ -58,3 +59,14 @@ func (sc Sidecar) GetVMID() (string, error) {
 	}
 	return vmid, nil
 }
+
+func (sc Sidecar) NetworkDataIsEmpty(network string) bool {
+	if sc.Networks[network].SubnetID == ids.Empty &&
```

Comment:
you can just return this

Reply:
maybe you can do sc.Networks[network] == NetworkData{}

Reply:
or, _, b := sc.Networks[network], b (if you desire to check if the network is a key)